packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Peering integration guide
+---
+
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+
+To join the Interledger network and be able to send and receive payments, you must add one or more peer(s) to your Rafiki instance. Peering establishes the connections needed for your Rafiki instance to interact with another account servicing entity (ASE). The purpose of this guide is to help you set up and manage peers.
+
+While this guide focuses on the conceptual and technical steps of adding and managing peers via the Backend Admin API, the Rafiki Admin application offers the same functionality in a user-friendly interface.
+
+Refer to the [Rafiki Admin user guide](/admin/admin-user-guide#peers) for detailed instructions and examples of creating and managing peers through the application.
+
+:::tip
+Whether you are using the Backend Admin API or the Rafiki Admin application, the underlying configurations and requirements remain the same. Choose the interface that best suits your individual workflow.
+:::
+
+## Prerequisites
+
+Before adding a peer, you and the account servicing entity you intend to peer with must both:
+
+- Run an implementation of an [Interledger connector](/integration/deployment/services/backend-service#interledger-connector) (ideally Rafiki).
+- Agree on an [asset](/overview/concepts/accounting#assets) for the peering relationship. You can set up multiple peering relationships with the same peer based on different assets. At least one asset shared by you and your peer must be added to your Rafiki instance prior to setting up the peering relationship. For more information, refer to [Assets](/integration/requirements/assets/).
+- Agree on a `maxPacketAmount` value, which specifies how many packets a payment is split into.
+- Communicate your respective static <LinkOut href="https://interledger.org/developers/rfcs/ilp-addresses/">Interledger (ILP) addresses</LinkOut>.
```

Comment:
Is this something that get assigned as part of Rafiki set up? If so, where would the ASE using this page find the address?

Reply:
ILP addresses are self-assigned and stored local to Rafiki as an environment variable. Updated wording here to reflect this. Worth considering adding additional information somewhere else on this page or maybe somewhere else completely.